web-ui/src/app/components/latest-blocks/latest-blocks.component.ts

```diff
@@ -70,9 +72,30 @@ export class LatestBlocksComponent implements OnInit, OnDestroy {
   }
 
   private onBlockRetrieved(response: Block[]) {
+    this.countNewestBlocks(response);
+    this.latestBlock = response[0];
     this.blocks = response;
   }
 
+  private countNewestBlocks(newBlocks: Block[]) {
+    if (!this.blocks) {
+      return;
+    }
+
+    const lastestBlockHash: string = this.latestBlock.hash;
```

Comment:
I would just pass this as an argument, as a hint, the `height` attribute from a block is just the block number and it is unique, it should be simpler to use that instead of the block hash, again, think on simplicity to test.